cub/cub/device/dispatch/dispatch_common.cuh

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#pragma once
+
+#include <cub/config.cuh>
+
+#if defined(_CCCL_IMPLICIT_SYSTEM_HEADER_GCC)
+#  pragma GCC system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_CLANG)
+#  pragma clang system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_MSVC)
+#  pragma system_header
+#endif // no system header
+
+CUB_NAMESPACE_BEGIN
+
+// Options for specifying memory aliasing
+enum class AliasOption
+{
+  MayAlias,
+  NoAlias
+};
```

Comment:
Suggestion: Maybe we can pick names that don't repeat the word `Alias`. Maybe just:
`MayAlias::yes` (and `no`).